Update documentation for v2.2 (#162)
ServiceNow/azimuth: docs/docs/getting-started/changelog.md

# Releases

## [2.2.0] - 2022-07-08

### Added

> - Contextual information has been added to most functionalities, linking directly to the relevant documentation link.

Reviewer: Detail, but I would avoir repeating "linking ... link".
```suggestion
- Contextual information has been added to most functionalities, linking directly to the relevant documentation.
```

- The F1 metric was added as a default metric.

### Changed

> - Smart tags are now grouped in families, so it is easier to parse them. This also impact how filters are displayed.

Reviewer: ```suggestion
- Smart tags are now grouped in families, so it is easier to parse them. This also impacts how filters are displayed.
```

Reviewer: And should we use a more human word than "parse"? Like
```suggestion
- Smart tags are now grouped in families, so it is easier to digest them. This also impacts how filters are displayed.
```

- The confusion matrix can be shown with raw values (not just normalized).

> - We now show the top 20, instead of 10, most similar examples in the utterances details.

Reviewer: ```suggestion
- We now show the top 20, instead of 10, most similar examples in the utterance details.
```
